### PR TITLE
Fix .greyscale() should be a method, not a property in Color extension

### PR DESCRIPTION
**Pull request by calvintam236**
Since `.greyscale()` is a shorthand of another method, and it is not a property, the extension should be `.greyscale()` instead of `.greyscale`.

### CODE REVIEW

**Review by WieFel — DISMISSED** [review]

**Comment by calvintam236** [comment]
@WieFel can you approve again after merging this changelog conflict?

**Review by WieFel — APPROVED** [review]

**Pull request merged**
